doc/man3/OSSL_CMP_exec_certreq.pod

```diff
@@ -157,6 +160,13 @@ The trust placed in it cannot be stronger than the trust placed in
 the I<oldwithold> certificate if present, otherwise it cannot be stronger than
 the weakest trust in any of the certificates in the trust store of I<ctx>.
 
+OSSL_CMP_get1_crlUpdate() uses a genm request message with infoType crlStatusList
+to obtain CRL from the CMP server referenced by I<ctx> in a genp response message
+with infoType crls. It uses oldcert referenced by I<ctx> and I<last_crl> to create
+request. On success it assigns to I<*crl> the CRL received.
```

Comment:
```suggestion
with infoType crls. It uses I<last_crl> and any `oldCert` referenced in I<ctx> to create
a request with a status field as described for L<OSSL_CMP_CRLSTATUS_create(3)>. 
On success it assigns to I<*crl> the CRL received.
```
Please list `L<OSSL_CMP_CRLSTATUS_create(3)>` also in the SEE ALSO section.

Reply:
Possibly this changes if the function gets a `cert` parameter as I suggest above.